src/tests/unit/tests/scanner/map-axe-tags-to-guidance-links.test.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as Axe from 'axe-core';
+import { link } from 'content/link';
+import { flatMap } from 'lodash';
+import { BestPractice, mapAxeTagsToGuidanceLinks } from 'scanner/map-axe-tags-to-guidance-links';
+
+describe('mapAxeTagsToGuidanceLinks', () => {
+    it('should map best-practice to the BestPractice link', () => {
+        expect(mapAxeTagsToGuidanceLinks(['best-practice'])).toEqual([BestPractice]);
+    });
+
+    it.each([null, undefined])('should map %p tags to no links', tags => {
+        expect(mapAxeTagsToGuidanceLinks(tags)).toEqual([]);
+    });
+
+    const wcagAAAtags = ['wcag223', 'wcag224', 'wcag248', 'wcag249', 'wcag325'];
+    const irrelevantAxeCoreTags = [
+        // axe-core specific, not required for our purposes
+        'cat.aria',
+        'experimental',
+        // refers to WCAG standard levels; we prefer to use guidance for specific wcag sections
+        'wcag2a',
+        'wcag2aa',
+        'wcag21a',
+        'wcag21aa',
+        // this WCAG section doesn't exist as of writing; if a future axe update picks up a new
+        // WCAG entry before we do, we want to omit it until we write new guidance for it
+        'wcag112',
+        // we intentionally omit AAA success criteria from guidance links; we don't want users to
+        // get confused about whether our tool supports a AAA assessment
+        ...wcagAAAtags,
+    ];
+    it.each(irrelevantAxeCoreTags)(
+        'should omit entries for irrelevant axe-core tag %s',
+        irrelevantTag => {
+            expect(mapAxeTagsToGuidanceLinks([irrelevantTag])).toEqual([]);
+        },
+    );
+
+    it('should omit entries for completely unrecognized tags', () => {
+        expect(mapAxeTagsToGuidanceLinks(['bogus'])).toEqual([]);
+    });
+
+    it.each`
+        tag           | expectedLink
+        ${'wcag111'}  | ${link.WCAG_1_1_1}
+        ${'wcag121'}  | ${link.WCAG_1_2_1}
+        ${'wcag1410'} | ${link.WCAG_1_4_10}
+    `('should map known wcag tag $tag to expected link', ({ tag, expectedLink }) => {
+        expect(mapAxeTagsToGuidanceLinks([tag])).toEqual([expectedLink]);
+    });
+
+    it('should handle multiple inputs in the list (omitting unrecognized cases)', () => {
+        expect(
+            mapAxeTagsToGuidanceLinks(['best-practice', 'wcag111', 'should-be-omitted']),
+        ).toEqual([BestPractice, link.WCAG_1_1_1]);
+    });
+
+    it('should sort the output', () => {
+        expect(mapAxeTagsToGuidanceLinks(['wcag111', 'wcag1411', 'best-practice'])).toEqual([
+            BestPractice,
+            link.WCAG_1_1_1,
+            link.WCAG_1_4_11,
+        ]);
+    });
+
+    const axe = Axe as any;
```

Comment:
Why do we do this instead of importing `* as axe from 'axe-core'`?

Reply:
Good catch! I copied this from the obsolete rule mapping tests verbatim, but your suggestion is both more concise and more type safe. Will update